src/sources/forus-webshop/js/angular-1/directives/VoucherCardDirective.js

```diff
@@ -1,11 +1,33 @@
 let VoucherCardDirective = function(
     $scope,
     VoucherService,
+    ModalService
 ) {
     $scope.voucherCard = VoucherService.composeCardData($scope.voucher);
 
     $scope.voucherCard.disabled = ($scope.voucherCard.type == 'product' && $scope.voucher.transactions.length) ||
         ($scope.voucher.expired && !$scope.voucher.transactions.length);
+
+    $scope.deleteVoucher = ($event, voucher) => {
```

Comment:
this function is a duplicate of deleteVoucher in VoucherComponent.js right?

Reply:
Yes, it actually is